tests/acceptance/00_basics/macros/if_triple.cf

```diff
@@ -0,0 +1,48 @@
+######################################################
+#
+#  Test that @if works with patch level
+#
+#####################################################
+
+body common control
+{
+      inputs => { "../../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+bundle common test
+{
+@if minimum_version(3.7.100)
+  classes:
+      "expected" expression => "any";
+@endif
+
+@if minimum_version(3.8.0)
+  classes:
+      "expected2" expression => "any";
+@endif
+
+@if minimum_version(2.100.0)
+  classes:
+      "expected_2_100" expression => "any";
+@endif
+
+@if minimum_version(3.8.300)
```

Comment:
You can write a secondary `"$(G.testfile).cf"` file with the right values inside, using `file_make()` in the `init` stage, and then run it from the primary test, looking for the right output.  See `tests/acceptance/10_files/09_insert_lines/nothing_to_repeat_error.cf` for an example of running a secondary test file.